documentation/visualization.md

```diff
@@ -115,5 +115,46 @@ To express that the dimension provides a specific _kind_ of data which is necess
 
 </aside>
 
+<section class="informative">
+
+##### Complex representation of time
+
+Representing temporal dimensions as literals typed as `xsd:date` or `xsd:dateTime` may be insufficient for complex scenarios. In such cases, data publishers may want to use the [Time Ontology in OWL](https://www.w3.org/TR/owl-time/) to represent temporal dimensions. This allows for the representation of complex temporal concepts, such as time periods, intervals, and repeating events, a well as joining cubes on the temporal dimension.
+
+When using Time Ontology in a cube, the values of a temporal dimension are resources of type `time:TemporalEntity`. The dimension should be expressed as a `time:GeneralDateTimeDescription` and have `Ordinal` scale type.
```

Comment:
@tpluscode see below at line 132 : 
it would be good to consistently reuse the already established `time:Interval` as in data i.e. at https://ld.admin.ch/time/month/1999-03  rather than `time:TemporalEntity` , don't you think ?

Reply:
Actually, this here may the the opposite side of the other comment. I write specifically about temporal entities because you may also have points in time represented as `time:Instant`. Surely, the Lindas examples use only intervals but the documentation is meant to be more general.

Well, in OWL Time instants are also intervals. Only proper interval is disjoint with instant. It is slightly weird to me

Reply:
ah, so to understand correctly: it's not of type `time:TemporalEntity` but could be of any subtype? - this would help to have mentioned to easier understand `time:TemporalEntity` by itself isn't attached directly ... but only through a subtype (or whichever wording you prefer...)